src/routes/orderRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const { orderController } = require('../controllers');
+const { verificateToken } = require('../middlewares')
```

Comment:
```suggestion
const { verifyToken } = require('../middlewares')
```
- verificate라는 단어가 없지 않나요?
- 팀내 공유하셔서 단어 적절한걸로 수정 부탁드립니다 :)